scenarios/appsec/test_customconf.py

```diff
@@ -57,7 +57,7 @@ def test_c04(self):
 
 
 # Basically the same test as Test_MissingRules, and will be called by the same scenario (save CI time)
-@released(java="0.93.0", nodejs="2.3.1", php_appsec="0.3.0", python="?", ruby="?")
+@released(java="0.93.0", nodejs="2.0.0", php_appsec="0.3.0", python="?", ruby="?")
```

Comment:
Did you tested this one? it's not tested on this CI, but is used on WAF rules repo